Writerside/topics/Quickstart-Up-And-Running.md

```diff
@@ -0,0 +1,528 @@
+# Quickstart: Running Your Game With Catena
+
+In this quickstart guide we will walk through how to set up and run your game on your own infrastructure using Catena Tools. 
+Through this guide you will set up a backend, basic fleet manager, along with a game client and game server. This is a __Static Fleet Management Solution.__ 
+This means you will have to manually provision servers, but they will report themselves to the backend.
+
+> This tutorial will use the following Catena Components to support a Multiplayer Workflow.
+>   * [Match Broker](Match-Broker.md)
+>   * [Server Manager](Server-Manager.md)
+>   * [Sidecar](Sidecar.md)
+>   * [Matchmaking](Matchmaking.md)
+>   * Infrastructure Templates
+
+## Overview
+
+By the end of this quickstart, you will be able to run your game, or a sample game in a multiplayer/online environment using AWS. Players will be able to:
+1. Log into your game
+2. Form a Party
+3. Request a Match
+4. Connect to a Server
+5. Play a game
+
+## Prerequisites
+
+What you will need before following the steps in this guide.
+
+* Access to Catena Tools
+* A Game with a client server model or the [Catena Lyra Demo](https://github.com/catenaTools/catena-lyra-demo)
+* The Catena Tools [AWS Infrastructure Template](https://github.com/CatenaTools/infrastructure/tree/main/aws/catena-core)
+* [Terraform](https://www.terraform.io) V1.71 or higher
+* An AWS account
+* Git bash (optional)
+* A domain name that you control and can configure in route53
+
+## Setup the Infrastructure
+
+Prior to running the Terraform code, set up a hosted zone for the domain that you will use. In the example, this will be `catenatools.com.`
+
+See [Creating a public hosted zone](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/CreatingHostedZone.html). The Terraform configuration will create a few subdomain routes on this url.
+
+Next, clone the `infrastructure` repo and open it in your favorite editor. We will be working out of the `aws/catena-core/` directory.
+
+**Git Bash**
+```bash
+git clone git@github.com:CatenaTools/infrastructure.git
+cd infrastructure/aws
+```
+
+This template sets up a working [dokku](https://dokku.com) instance on the domain you select, along with a few additional settings to simplify deploying and running the platform, including an IAM role for `aws ssm` logins and an Elastic IP.
```

Comment:
Maybe add another line explaining why we use Dokku?

This Catena deployment utilizes Dokku to make deploying updates as simple as `git push`. You don't need to worry too much about all of the infrastructure if you don't want to. This tutorial will outline the pieces that are important for you to understand in order to get up and running.